fecgov/fec-cms
# added new oig category and updated names of 2 categories

## Summary (required)

- Resolves #4959 

Added new category: Management challenges reports
Rename "Other reports and plans - OIG" to "Other OIG reports and plans"
Rename "Strategic plans - OIG" to " OIG strategic plans"

### Required reviewers

1 content, 1 UX, 1 front end

## Impacted areas of the application

General components of the application that this PR will affect:

-  OIG reports page

## Screenshots

<img width="841" alt="Screen Shot 2021-12-01 at 11 34 11 AM" src="https://user-images.githubusercontent.com/12799132/144274901-0088caa0-f944-4483-9d10-607439da1578.png">

## How to test

Check to make sure the new categories are there

- Content team: Look at the screenshot
- Frontend and UX team:
   - `cd fec && ./manage.py migrate`
   - `./manage.py runserver`
   - Go to the [OIG reports](http://localhost:8000/admin/pages/7280/) and update a report to have the new "Management challenges reports" category.
